builtin/game/falling.lua

```diff
@@ -18,27 +18,37 @@ core.register_entity(":__builtin:falling_node", {
 	},
 
 	node = {},
+	meta = {},
 
-	set_node = function(self, node)
+	set_node = function(self, node, meta)
 		self.node = node
+		if meta ~= nil then
+			self.meta = meta
+		else
+			self.meta = {}
+		end
 		self.object:set_properties({
 			is_visible = true,
 			textures = {node.name},
 		})
 	end,
 
 	get_staticdata = function(self)
-		return core.serialize(self.node)
+		local ds = {
+			node = self.node,
+			meta = self.meta,
+		}
+		return core.serialize(ds)
 	end,
 
 	on_activate = function(self, staticdata)
 		self.object:set_armor_groups({immortal = 1})
 		
-		local node = core.deserialize(staticdata)
-		if node then
-			self:set_node(node)
+		local ds = core.deserialize(staticdata)
+		if ds and ds.node then
+			self:set_node(ds.node, ds.meta)
```

Comment:
nodes that started falling before this change will be lost. Not sure how worth it is to check for this